stacks/camel/1.0.0/devfile.yaml

```diff
@@ -0,0 +1,78 @@
+schemaVersion: 2.2.0
```

Comment:
You can use a newer version; at least 2.2.2
The latest version is 2.3.0, @michael-valdron do you know if we can use 2.3.0 schema version for stacks?